### DISCUSSION TITLE
torch problem

### DISCUSSION THREAD

**Discussion by m11113040** [open-edge-platform/anomalib#1718] [Q&A]
hi everyone,i have a problem in train custom data,I already install torch 2.2.0,but still told me no install torch
![image](https://github.com/openvinotoolkit/anomalib/assets/157609224/ee53f61f-0522-4348-9fd5-c4a382d764dd)

**Comment by samet-akcay** [comment]
Hi @m11113040, I think the error might be related to the path to your config file. Please make sure that the path of your config file is correct. Looks like you are on the `desktop` directory. Can you check if you have `src/configs/data` directories under the `desktop` directory?

**Reply by m11113040** [reply]
i change my anomalib file path,but still error
![image](https://github.com/openvinotoolkit/anomalib/assets/157609224/0febb9af-5247-45d2-b9cc-b4702b8ece16)
![image](https://github.com/openvinotoolkit/anomalib/assets/157609224/1be69477-6a03-49cc-a494-863e519a43b8)

**Reply by samet-akcay** [reply]
It might be safer to run the command from anomalib root. I see that you are on the `src/config` directory.
![305306542-0febb9af-5247-45d2-b9cc-b4702b8ece16](https://github.com/openvinotoolkit/anomalib/assets/17574420/e98989c8-4cbd-4953-9213-d50750315994)

**Reply by m11113040** [reply]
![image](https://github.com/openvinotoolkit/anomalib/assets/157609224/8f99a922-bf12-41bf-90cc-e165791e5bf8)
is my classification.yaml file has problem?
I just follow (https://anomalib.readthedocs.io/en/latest/markdown/guides/how_to/data/custom_data.html) to create a classification.yaml file,is that correct step? Or i have to do other step? thank
